client/ide/lib/realtimemanager.coffee

```diff
@@ -15,6 +14,7 @@ module.exports = class RealtimeManager extends KDObject
     @collaborativeInstances = []
     @collaborativeEventListeners = {}
 
+    GoogleApiClient  = require './googleapiclient'
```

Comment:
Main reason to move this into the constructor of `RealtimeManager` is [this](https://github.com/koding/koding/blob/master/client/ide/lib/googleapiclient.coffee#L71-L84) block of code. 

I did this because we are already waiting for its `ready` event to execute rest of the code.